include/dbclean.php

```diff
@@ -16,22 +16,48 @@
 load_config('config');
 load_config('system');
 
-update_shadow_copy();
+remove_orphans();
```

Comment:
What does "actions" mean in the context of the PSR?

BTW: I'm really, really bad in understanding formal written descriptions - especially when they aren't written in German.